app-modules/knowledge-base/config/roles/web/knowledge_base_management.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+return [
+    'model' => [
+        'knowledge_base_item' => [
+            '*',
+        ],
+        'knowledge_base_category' => [
+            '*',
+        ],
+        'knowledge_base_quality' => [
+            '*',
+        ],
+        'knowledge_base_status' => [
+            '*',
+        ],
```

Comment:
These field property permissions should actually be separated away from the base Model CRUD access. As we will have Users that can access the KB for example, but they are not admins that should be able to change these field settings.

@dgoetzit what do you think about the naming for this separation? `knowledge_base_management` and `knowledge_base_field_settings_management`?

We will need to do this also for others like Prospects and Service Requests.